include/deal.II/base/parameter_handler.h

```diff
@@ -1654,6 +1673,13 @@ class ParameterHandler : public Subscriptor
    */
   std::unique_ptr<boost::property_tree::ptree> entries;
 
+  /**
+   * A map that stores whether an entry added to the parameter handler has
+   * been set by any of the functions parsing input parameters or by a set
+   * function of this class.
+   */
+  std::map<std::string, std::pair<bool, bool>> entries_set_status;
```

Comment:
can you explain what the second part of the map is?

Reply:
Sorry, I have not seen this question so far. I adjusted the comment.